specification/azurefleet/resource-manager/Microsoft.AzureFleet/stable/2024-11-01/azurefleet.json

```diff
@@ -0,0 +1,3581 @@
+        },
+        "localStorageDiskTypes": {
+          "type": "array",
+          "description": "The local storage disk types specified as a list. LocalStorageSupport should be set to \"Included\" or \"Required\" to use this VMAttribute. \nIf localStorageSupport is \"Excluded\", this VMAttribute can not be used.",
+          "items": {
+            "$ref": "#/definitions/LocalStorageDiskType"
+          },
+          "x-ms-identifiers": []
+        },
+        "dataDiskCount": {
+          "$ref": "#/definitions/VMAttributeMinMaxInteger",
+          "description": "The range of data disk count specified from Min to Max. Optional parameter. Either Min or Max is required if specified."
+        },
+        "networkInterfaceCount": {
+          "$ref": "#/definitions/VMAttributeMinMaxInteger",
+          "description": "The range of network interface count specified from Min to Max. Optional parameter. Either Min or Max is required if specified."
+        },
+        "networkBandwidthInMbps": {
+          "$ref": "#/definitions/VMAttributeMinMaxDouble",
+          "description": "The range of network bandwidth in Mbps specified from Min to Max. Optional parameter. Either Min or Max is required if specified."
+        },
+        "rdmaSupport": {
+          "$ref": "#/definitions/VMAttributeSupport",
+          "description": "Specifies whether the VMSize supporting RDMA (Remote Direct Memory Access) should be used to build Fleet or not."
+        },
+        "rdmaNetworkInterfaceCount": {
+          "$ref": "#/definitions/VMAttributeMinMaxInteger",
+          "description": "The range of RDMA (Remote Direct Memory Access) network interface count specified from Min to Max. Optional parameter. Either Min or Max is required if specified."
+        },
+        "acceleratorSupport": {
+          "$ref": "#/definitions/VMAttributeSupport",
+          "description": "Specifies whether the VMSize supporting accelerator should be used to build Fleet or not."
+        },
+        "acceleratorManufacturers": {
+          "type": "array",
+          "description": "The accelerator manufacturers specified as a list. acceleratorSupport should be set to Included or Required to use this VMAttribute.",
+          "items": {
+            "$ref": "#/definitions/AcceleratorManufacturer"
+          },
+          "x-ms-identifiers": []
+        },
+        "acceleratorTypes": {
+          "type": "array",
+          "description": "The accelerator types specified as a list. acceleratorSupport should be set to Included or Required to use this VMAttribute.",
+          "items": {
+            "$ref": "#/definitions/AcceleratorType"
+          },
+          "x-ms-identifiers": []
+        },
+        "acceleratorCount": {
+          "$ref": "#/definitions/VMAttributeMinMaxInteger",
+          "description": "The range of accelerator count specified from min to max.. Optional parameter. Either Min or Max is required if specified."
```

Comment:
acceleratorSupport should be set to "Included" or "Required" to use this VMAttribute. \nIf acceleratorSupport is "Excluded", this VMAttribute can not be used." should be added to description.

Reply:
Updated.